src/UserService.ts

```diff
@@ -132,6 +137,17 @@ class UserService extends Base {
       },
     );
   }
+
+  async getBillingPageToken({ username }: { username: string }): Promise<BillingPageToken> {
+    const { apiEndpointAddress, fetch } = this.lensPlatformClient;
+    const url = `${apiEndpointAddress}/users/${username}/billing-page-token`;
+    const json = await throwExpected(
+      async () => fetch.get(url),
+      { 404: () => new NotFoundException(`Billing page token for ${username} not found`) },
```

Comment:
Is 404 the only type of exception we will get from this endpoint?

Reply:
I think not. Waiting for implementation from @nevalla to add all relevant exceptions. I suppose we need at least 403 here as well.

Reply:
Added a TODO list to PR description to reflect this.

Reply:
FYI 403 is automatically mapped to `ForbiddenException`, but it's fine to have it here if we want to customize the Error message.